.eslintrc.yml

```diff
@@ -6,6 +6,10 @@ env:
   browser: true
 
 settings:
+  import/ignore:
+    # Ignores blind export of everything from node_modules as well as it's done
+    # by default
+    - design/index.js$
```

Comment:
I don't understand why you added this here. I read about the [rule](https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-unresolved.md#ignore) but it didn't make any sense to me why it had to be included here. Did you experience any issues with importing `desing`?

Reply:
Yes, I experienced issues with it. It ignores `node_modules`. So, if I import with wildcard (`*`) from `node_modules`, the linter cannot resolve anything and throws an error.

Reply:
I see... Nonetheless, I tried it without it and it didn't throw any error (it's the default behaviour as you wrote in a comment, isn't it?). I guess these lines are not necessary, are they?

Reply:
It does not throw because in this PR contains no import from `design`

Reply:
What about this https://github.com/oacore/dashboard/blob/alias/design/index.js#L1?

I also tried to change it like this but again no error was thrown:
```
export * from '@oacore/design'
import * as D from '@oacore/design'
```

Reply:
Try to merge [this}(https://github.com/oacore/dashboard/blob/alias/components/layout/app-bar.jsx#L2) with line 7 in the same file and test again.

Reply:
Ohh I get it now. Thanks. It is a shame it doesn't work very well (IDE shows me a warning). This is another win for typescript. 😄